static/js/components/email/AutomaticEmailOptions.js

```diff
@@ -0,0 +1,55 @@
+// @flow
+import React from 'react';
+import { RadioButton, RadioButtonGroup } from 'material-ui/RadioButton';
+
+import {
+  ONE_TIME_EMAIL,
+  EMAIL_CAMPAIGN
+} from './constants';
+
+export default class EmailCompositionType extends React.Component {
+  props: {
+    automaticEmailType:    ?string,
```

Comment:
as I mentioned below, I think here we should just pass down `sendAutomaticEmails: boolean`